jobs/integration/validation.py

```diff
@@ -379,18 +379,21 @@ async def query_dashboard(url, config):
     # get k8s version
     app_config = await model.applications["kubernetes-control-plane"].get_config()
     channel = app_config["channel"]["value"]
-    # if we do not detect the version from the channel eg edge, stable etc
-    # we should default to the latest dashboard url format
-    k8s_version = (2, 0)
-    if "/" in channel:
-        version_string = channel.split("/")[0]
-        k8s_version = tuple(int(q) for q in re.findall("[0-9]+", version_string)[:2])
+    dash_ns = "kubernetes-dashboard"
+    if "latest" not in channel:
+        # if we do not detect the version from the channel eg edge, stable etc
+        # we should default to the latest dashboard url format
+        k8s_version = (2, 0)
+        if "/" in channel:
+            version_string = channel.split("/")[0]
+            k8s_version = tuple(
+                int(q) for q in re.findall("[0-9]+", version_string)[:2]
+            )
+
+        if k8s_version < (1, 16):
+            dash_ns = "kube-system"
```

Comment:
Or should we just forget about testing k8s versions < `1.16`?

```suggestion
    dash_ns = "kubernetes-dashboard"
```